### ISSUE TITLE
UI Problems

### ISSUE THREAD

**Issue by alexander255** [Cookie-AutoDelete/Cookie-AutoDelete#62]
After having tried CAD for a couple of minutes there's a couple of minor and not-so-minor UI issues I'd like to have fixed. Since most of this stuff is highly specific I'll fix them myself when I have time (probably in about 2 weeks) and will just collect my observations in your bug tracker in the mean time:

  * Mark UI elements as non-selectable (preferences & browserAction, expect for list of recently deleted cookies)
  * Use the arrow cursor (vs the text selection cursor) consistently in those places as well
  * Fix display of browserAction with dark themes:
![spectacle x21003](https://user-images.githubusercontent.com/246386/27486970-92313c64-5832-11e7-9100-0cb541731d97.png)
  * Maybe make font a bit less tiny?
  * Fix text color of numbers on the settings page: ![screenshot_20170623_164317](https://user-images.githubusercontent.com/246386/27487119-125ac612-5833-11e7-96c2-e280d9b324d1.png) (also dark theme related probably)

**TL;DR**: Please just ignore this, it's a brain-dump of stuff I noticed after deciding on and switching to CAD. I'll let code speak soon-ish.

**Comment by mrdokenny** [comment]
> I'll fix them myself when I have time (probably in about 2 weeks) and will just collect my observations in your bug tracker in the mean time

No problem. I'll have an Development workflow section on the main page soon.

> Fix display of browserAction with dark themes:

I think this might be similar to #31, but I don't really use linux all that much, so even though I commented on a possible fix, I can't really debug and test it.

**Comment by ke-d** [comment]
Closed for the release of 2.X.X
